pkg/controller/util.go

```diff
@@ -68,7 +68,7 @@ const (
 )
 
 var snapshotterSecretParams = deprecatedSecretParamsMap{
-	name: "Snapshotter",
+	name:                         "Snapshotter",
```

Comment:
I think this change will cause "go fmt" to fail in 1.10.3.

Reply:
ah.. It looks like thats causing it. I have removed this change for now.